### ISSUE TITLE
Gene-Gene Network: Explain the algorithms being used to rank the genes 

### ISSUE THREAD

**Issue by dongbohu** [greenelab/adage-server#295]
With the new slider bar that controls the max number of genes displayed in the network, a natural question from users will be: how are the genes in the network ranked? 

So far `d3.network.js` has two algorithms to rank the genes. But it is not trivial to explain them in a few sentences. The best place to describe the ranking algorithms is probably the `help` page, and the gene-gene network web UI should have a link that points to this description.

**Comment by cgreene** [comment]
@dongbohu : can you point me towards the point in the source code where these are defined. I can try to write some text.

**Comment by dongbohu** [comment]
@cgreene: See lines 60-92 of this file:
https://github.com/aaronkw/d3-network/blob/master/d3.network.js

**Comment by cgreene** [comment]
Ok - here's the summary:

```
Some genes are connected to many genes. Other genes are connected to few genes. Without background correction, the network view prioritizes the genes that are most connected to the query genes. With the optional background correction step, the network view prioritizes genes that are more connected to the query genes than to other genes.
```

My impression is that this selection isn't going to be user facing for us right now though - is that right? I am under the impression that the ADAGE server's default behavior is non-background corrected (in contrast with GIANT, where the default is background corrected).

**Comment by dongbohu** [comment]
@cgreene: Yes, right now the ADAGE server's default behavior is non-background corrected. Do you think version 1.0 should allow users to enable/disable this option?

**Comment by cgreene** [comment]
Hmm - may be confusing. The background correction was added to GIANT based on user feedback, if I recall correctly. I think we debated making it optional and decided not to because it was thought that it'd be too confusing. The same thing is probably true here. If we get users complaining that "the same gene shows up for every query" then we may want to consider enabling it. But these networks may be different enough from GIANT that this is less of a problem, so my inclination is to go with off unless we get some requests.

We should probably put this text somewhere as a comment in the code though. This way if we ever do enable it, we have it close at hand :)

**Comment by dongbohu** [comment]
Let me add the summary above in `d3.network.js` above and link this issue's URL as well.

**Comment by dongbohu** [comment]
@mhuyck and @rzelayafavila: I addressed most of your comments. @mhuyck: for your comment on lines 216-223, I proposed one resolution. If you agree, I will implement it. Thanks.

**Comment by dongbohu** [comment]
Fixed in PR #342.
